### DISCUSSION TITLE
please, fix issue me

### DISCUSSION THREAD

**Discussion by nhatbao123** [strapi/strapi#7270] [Legacy Topics]
I installed CONTENT MANAGER already so why is it still not shown in the plugins section.
![Annotation 2020-07-31 021808](https://user-images.githubusercontent.com/68341519/88964858-2bebe100-d2d4-11ea-8697-56492e0c4387.png)

**Comment by lauriejim** [comment]
Here is the answer - https://github.com/strapi/strapi/issues/5974

**Reply by nhatbao123** [reply]
thank very much !!
